datasafe-directory/datasafe-directory-impl/src/main/java/de/adorsys/datasafe/directory/impl/profile/operations/actions/ProfileRetrievalServiceImpl.java

```diff
@@ -24,7 +24,7 @@
 @RuntimeDelegate
 public class ProfileRetrievalServiceImpl implements ProfileRetrievalService {
 
-    private final DFSConfig dfsConfig;
+    protected final DFSConfig dfsConfig;
```

Comment:
There is no need to change visibility of this field, simply add extra field to extending class